scripts/publish-docs.js

```diff
@@ -0,0 +1,20 @@
+const shelljs = require('shelljs');
+
+if (!process.env.TRAVIS_TAG) {
+  throw new Error('This script can only be executed when a git tag is created');
+}
+
+if (process.env.TRAVIS === 'true') {
+  const remoteUrl = `https://${
+    process.env.GH_TOKEN
+  }@github.com/commercetools/ui-kit.git`;
+  shelljs.exec('git config --global user.email "npmjs@commercetools.com"');
+  shelljs.exec('git config --global user.name "ct-release-bot"');
+  shelljs.exec(`git remote add origin-docs ${remoteUrl} > /dev/null 2>&1`);
+}
+
+shelljs.exec('git checkout docs-production');
```

Comment:
Nitpick but a thought. Maybe flip this around. We have `master` which we happily merge to. Then we have a `latest/docs` branch. To align naming with the npm channels and also be able to more nicely scope them. So we can have `latest/docs` `latest/build` (if that makes sense). Just an idea.

Reply:
I'm sure it makes sense but I'm not following you right now 😅

Reply:
Can you elaborate please?

Reply:
It's me being complicated. Sorry. I am only suggesting to rename the branch to `latest/docs`. It has the following advantages:

1. It's nicer to view in some Git UIs (as they treat slashes as folders)
2. It's easier to align/correlate mentally to the `@latest` tag in npm
3. It's easier to extend if we have more branches which contain other things: dist files, docs etc.

Reply:
Ok got it. Yes, makes sense to me!

I'll rename things 👌 Thanks for the suggestion 🎉

Reply:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/1110551/45495673-0010f200-b774-11e8-92d7-71ca55e0fc24.png">

and https://github.com/commercetools/ui-kit/pull/45/commits/3c4c15e7cf82c4144b86c38f9cff8b88ec378ee6